applications/asset_tracker_v2/src/modules/Kconfig.data_module

```diff
@@ -169,6 +169,14 @@ config DATA_SAMPLE_NEIGHBOR_CELLS_DEFAULT
 	  data in sample requests sent to other modules. This configuration can be overwritten by
 	  changing the application's real-time configuration using the cloud-side state.
 
+config DATA_SAMPLE_WIFI_DEFAULT
```

Comment:
This has been left out intentionally because WiFi sampling has been kind of assumed to be wanted when WiFi is built into the image. So it's different in nature compared to GNSS and neighbor_cell which are always available in the build.

Reply:
Let's add it for consistency although it's quite useless and similar Kconfig's for GNSs and cellular could be even removed.

Reply:
This KConfig has to be added to document though.